modules/common/src/main/java/org/eclipse/xpanse/common/openapi/OpenApiUrlManage.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+package org.eclipse.xpanse.common.openapi;
+
+import java.net.InetAddress;
+import java.net.UnknownHostException;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder;
+
+/**
+ * Bean to provide helper methods for generating OpenAPI files.
+ */
+@Slf4j
+@Component
+public class OpenApiUrlManage {
+
+    private final String openapiPath;
+    private final Integer port;
+
+    /**
+     * OpenApiUtil constructor.
```

Comment:
should be: OpenApiUrlManage constructor.

Reply:
fixed